Add 6 icons and links

leojue opened the pull request:
## Description

<!-- 
Please include a summary of the change. Please also include relevant motivation and context.

If this PR is an icon addition one, please provide a short summary on what icons you added, changed, or linked
-->

I recently discovered lawnicons and I'm amazed by its beautiful simplicity. And - finally the Material You icons are making sense! So I wanted to contribute a few icons that are currently missing.
I hope that they are adhering to the design guidelines. If anything should be changed, please tell me so. In my oppinion the most difficult icon is the one from _ING_...

## Type of change
<!-- Replace :x: with :white_check_mark: to "check" the specified bullet -->

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories, such as copyediting)

<!-- Erase the below text if you are not making an icon addition -->
## Icons addition information

### Icons added
* Azure (`com.microsoft.azure`)
* Bose Connect (`com.bose.monet`)
* Calm (`com.calm.android`)
* Crewmeister (`com.crewmeister.employeeapp`)
* ING (`de.ingdiba.bankingapp`)
* Scalable Capital (`capital.scalable.droid`)

### Icons linked
* DJI (linked `dji.go.v4` to `@drawable/dji`)
* ING (linked `com.ingbanktr.ingmobil` to `@drawable/ing`)
* ING (linked `com.ing.mobile` to `@drawable/ing`)
* ING (linked `ro.ing.mobile.banking.android.activity` to `@drawable/ing`)
* ING (linked `www.ingdirect.nativeframe` to `@drawable/ing`)
* ING (linked `com.ing.banking` to `@drawable/ing`)
* ING (linked `de.ingdiba.bankingapp` to `@drawable/ing`)
* ING (linked `com.comarch.security.mobilebanking` to `@drawable/ing`)
* ING (linked `it.ingdirect.app` to `@drawable/ing`)
* ING (linked `com.ing.lu.mobile` to `@drawable/ing`)
* ING (linked `lu.ing.mying.app` to `@drawable/ing`)
* ING (linked `lu.ing.myingpro` to `@drawable/ing`)

x9136 commented:
Hey!

You can check [Lawnicon Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md#icon-guidelines) by yourself. TL;DR: use only strokes, main thickness should be 12px, make sure that icons fit the 160×160px content area and all angles are rounded.

This means that all icons require different changes.

As you can see in **Files changes**, scalable_capital.svg is already here. For Bose Connect you'll have to use a square 144×144px from the guidelines. ING is really complicated, but there is no restriction in the guidelines that icons should be completely similar to the originals. It is worth simplifying the details in ING icon.

leojue commented:
Hey @x9136,

thanks for the detailed answer! Here are the things I changed following your recommendations:

**`Scalable Capital`**: Reverted change 
**`Crewmeister` and `Azure`**: Adapted so that they should adhere to the design guidelines (stroke width 12px)
**`Calm`**: Adapted to 6px stroke width because of text
**`Ing`**: Tried out new design (credit: https://www.behance.net/gallery/102483707/ING-Rebranding). Is the difference to the original icon to big?

Apart from that I am not shure what changes you are proposing for the Bose Connect logo, because it already fits the 144x144px square. Could you please elaborate on that?

If there is anything else that should be changed please let me know!

x9136 commented:
@leojue you still use fill instead of strokes and the content area is wrong. I did an example.
<img width="933" alt="Screenshot 2023-07-30 at 14 43 47" src="https://github.com/LawnchairLauncher/lawnicons/assets/60105060/b9f885c6-70d4-475c-8056-d583cfe5bf1a">

>Apart from that I am not shure what changes you are proposing for the Bose Connect logo, because it already fits the 144x144px square. Could you please elaborate on that?

I'm not sure if it would work to leave Bose Connect like this. It would be great if it does, but the mods can clarify that. In my mind this icon would be appropriate if it was like this:
<img width="229" alt="Screenshot 2023-07-30 at 14 44 01" src="https://github.com/LawnchairLauncher/lawnicons/assets/60105060/6b3d64c1-47fa-4f8c-a37d-cd51bdcdcb5d">

leojue commented:
> you still use fill instead of strokes and the content area is wrong.

Sorry about that! I fixed the content area. And about the strokes: I used Adobe Illustrator and made this icon out of strokes. But the only way I know to apply the 6px rounded corners is to make contour lines out of the strokes and then apply the rounded corners.

> I prefer to create my own version instead of using someone else's redesign. I don't know what to tell you about that.

I completely agree with that. I just thought that it exceeded my abilities to create a good version of this icon (and I tried...). But now I'm going to try making my own version. 

> I'm not sure if it would work to leave Bose Connect like this. It would be great if it does, but the mods can clarify that. In my mind this icon would be appropriate if it was like this:

I completely see your point and am not sure either. I chose the more minimal version because of how the icon looks like on my phone:
![WhatsApp Image 2023-07-30 at 14 21 26](https://github.com/LawnchairLauncher/lawnicons/assets/63466657/cacd44ef-65ec-4bc7-b92f-779f9b507c30)
![WhatsApp Image 2023-07-30 at 14 21 26 (1)](https://github.com/LawnchairLauncher/lawnicons/assets/63466657/afa89df6-725a-4707-9522-4e9754c2b66e)
I thought that this would stay true to the original. But I also like your version and would be happy to hear a clarification.

Chefski commented on svgs/azure.svg:
The icon cannot use a fill style. Please change it to only use outlines. Please make sure it fits the 160x160 content are too

Chefski commented on svgs/ing.svg:
The icon uses a fill style, please change it to an outline style. I'd suggest looking at the `brave_browser` icon for some help.

Chefski commented on svgs/calm.svg:
The lines seem to be varying in width. Could you try use stroke that uses only one width like 6px?

Chefski commented on svgs/bose_connect.svg:
Unfortunately this icon won't work. As suggested by @x9136, you can put the plus inside a square. Squares have a content area of 144x144 instead of 160x160.
